docs/functions.json

```diff
@@ -1448,6 +1448,25 @@
             }
         ]
     },
+    {
+        "signature": "week_number(date)",
```

Comment:
we have other functions `number()` and `format_number()` so changed this from `week_num`